includes/widgets/wsl.auth.widgets.php

```diff
@@ -191,8 +191,19 @@ function wsl_render_auth_widget( $args = array() )
 				}
 			}
 
+			// Use the provided redirect_to if it is given and this is the login page.
+			if ( in_array( $GLOBALS['pagenow'], array( 'wp-login.php', 'wp-register.php' ) ) &&
```

Comment:
it make sense. I just have one tiny nitpicking point: it would be better to take that `if` statement out of the `foreach` which may save few µs, I think.